saleor/plugins/manager.py

```diff
@@ -79,8 +79,12 @@ def _load_plugin(
 
         return PluginClass(configuration=plugin_config, active=active, channel=channel)
 
+    def __eq__(self, other):
```

Comment:
Why do we need this?

Reply:
Because test `test_release_dangling_unfinished_payment_task` faile on `release_checkout_payment.assert_called_once_with(payment_dummy, manager)`

Reply:
Hhm... I don't think that this condition is correct. It will always return a true for managers with the same plugins, but the configuration of them can be different, right?  Manager stores plugins per channel, so one manager can have active two plugins in 3 channels, and second manager has two plugins in 1 channel, and this condition will return true. 
Also, I am not convinced that we should modify the base code, just for unittests.  I think that for your case you should mock a given action on the manager and then check it was called with proper values.  Something like this:
`@patch("saleor.plugins.manager.PluginsManager.notify")`. If payment should be refunded then you mock refund action, for void you mock manager.void action. You can also check if the function was called with an instance of PluginManager.

Reply:
OK, I've removed changes in manager and use ANY. `release_checkout_payment.assert_called_once_with(payment_dummy, ANY)`